ui/packages/consul-ui/app/instance-initializers/container.js

```diff
@@ -3,38 +3,33 @@ import require from 'require';
 import merge from 'deepmerge';
 
 const doc = document;
-const appName = 'consul-ui';
-const appNameJS = appName
-  .split('-')
-  .map((item, i) => (i ? `${item.substr(0, 1).toUpperCase()}${item.substr(1)}` : item))
-  .join('');
 
 export const services = merge.all(
-  [].concat(
-    ...[...doc.querySelectorAll(`script[data-${appName}-services]`)].map($item =>
-      JSON.parse($item.dataset[`${appNameJS}Services`])
-    )
+  [...doc.querySelectorAll(`script[data-services]`)].map($item =>
+    JSON.parse($item.dataset[`services`])
   )
 );
 
 const inject = function(container, obj) {
   // inject all the things
   Object.entries(obj).forEach(([key, value]) => {
-    switch(true) {
-      case (typeof value.class === 'string'):
-        if(require.has(value.class)) {
-          container.register(key.replace('auth-provider:', 'torii-provider:'), require(value.class).default);
+    switch (true) {
+      case typeof value.class === 'string':
+        if (require.has(value.class)) {
+          container.register(
+            key.replace('auth-provider:', 'torii-provider:'),
+            require(value.class).default
+          );
         } else {
           throw new Error(`Unable to locate '${value.class}'`);
         }
-      break;
+        break;
     }
   });
-}
+};
```

Comment:
In this PR: Yey `prettier`. Nothing to see here, please move on, do not pass go, do not collect $200 😄 

Previous to this PR: This is where we do:

```javascript
// container here is pretty much the same as the ember `application`
container.register('component:consul/partition/selector', 'consul-ui/components/consul/partition/selector');
```

We do an extra replacement to remove the word `torii` from our application config (we might switch that out at some point _not_ to use the `torii` addon)